templates/zerver/app/index.html

```diff
@@ -37,9 +37,20 @@
     #app-loading-middle-content h3 {
     text-align: center;
     }
+    @keyframes fadeIn {
+    99% {
+    visibility: hidden;
+    }
```

Comment:
Changing `visibility` from `hidden` to `hidden` is redundant; just delete this frame.

Reply:
We are trying to hold the value of `visibility` as `hidden` for as long as possible. See https://stackoverflow.com/a/30856455